frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -2,18 +2,21 @@ import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
 import { K8sResourceKind } from '@console/internal/module/k8s';
 
-const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephStorageProvisioners = [
+  'ceph.rook.io/block',
+  'openshift-storage.cephfs.csi.ceph.com',
+  'openshift-storage.rbd.csi.ceph.com',
+];
 const cephStorageLabel = 'cluster.ocs.openshift.io/openshift-storage';
 
 export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
 
-export const getCephPVCs = (pvcsData: K8sResourceKind[] = []): K8sResourceKind[] =>
-  pvcsData.filter((pvc) =>
-    cephStorageProvisioners.includes(
-      _.get(pvc, 'metadata.annotations["volume.beta.kubernetes.io/storage-provisioner"]'),
-    ),
-  );
+export const getCephPVCs = (
+  cephScNames: string[] = [],
```

Comment:
```suggestion
  cephSCNames: string[] = [],
```
SC would be more descriptive.